static/src/stylesheets/module/content/_content.scss

```diff
@@ -724,6 +724,15 @@
     .from-content-api {
         @include f-textSans;
     }
+    .element.element-pullquote{
+      .pullquote-paragraph {
+        font-family: $f-sans-serif-headline;
+        color: $paid-article-brand;
+      }
+    }
+    a.in-body-link--immersive {
```

Comment:
Yes, I know. But do you need to be more specific than just `.in-body-link--immersive`? What other styles do you want to override?